receiver/sqlserverreceiver/metadata.yaml

```diff
@@ -86,6 +86,15 @@ metrics:
     gauge:
       value_type: double
     extended_documentation: This metric is only available when running on Windows.
+  sqlserver.lock.wait.count:
+    enabled: false
+    description: Cumulative count of lock waits that occurred.
+    unit: "{waits}"
```

Comment:
I'd appreciate feedback on what the unit should be here, if anyone has any thoughts.

Reply:
It should be `{wait}` as described in https://github.com/open-telemetry/semantic-conventions/blob/v1.33.0/docs/general/metrics.md#instrument-units.

> All non-units that use curly braces to annotate a quantity need to match the grammatical number of the quantity it represent. For example if measuring the number of individual requests to a process the unit would be `{request}`, not `{requests}`.

Reply:
Thanks for calling this out @andrzej-stencel, updated.